src/containers/offer-page/chat-dialog-window/ChatDialogWindow.tsx

```diff
@@ -89,14 +101,40 @@ const ChatDialogWindow: FC<ChatDialogWindow> = ({ chatInfo }) => {
     messagesListWithDate
   )
 
-  const questionsList = questions.map((el) => (
-    <Typography key={el.question} sx={styles.question}>
-      {t(el.question)}
-    </Typography>
-  ))
+  const questionsList = creatingChat ? (
+    <Box sx={styles.chatCreateBox}>
+      <Loader size={20} />
+      <Typography sx={styles.question}>{t('chatPage.creating')}</Typography>
+    </Box>
+  ) : (
+    questions.map((el) => (
+      <Typography key={el.question} sx={styles.question}>
+        {t(el.question)}
+      </Typography>
+    ))
+  )
 
   const closeChatWindow = () => setChatInfo(null)
 
+  const openChatInNewTab = (chatId: unknown) => {
+    localStorage.setItem('currentChatId', chatId as string)
+    window.open(authRoutes.chat.path, '_blank', 'noopener noreferrer')
+  }
+
+  const handleCreateNewChat = async () => {
+    setCreatingChat(true)
+    const newChat = await createChat()
+    openChatInNewTab(newChat.data._id)
+    setCreatingChat(false)
+  }
+
+  const handleRedirectToChat = async () => {
+    const chatId: unknown = chatInfo.chatId
```

Comment:
i think it will be better to don't use `unknown` type and there is no need to type constant variable that get value from typed object

Reply:
> i think it will be better to don't use `unknown` type and there is no need to type constant variable that get value from typed object

yeah, that would be better. Fixed it 👍